batch-nft-reveal/package.json

```diff
@@ -6,15 +6,17 @@
   "scripts": {
     "compile": "hardhat compile",
     "test": "hardhat test",
-    "deploy": "hardhat run scripts/deploy.ts --network",
+    "deploy": "hardhat deploy --network",
     "flatten": "bash bin/flattener.sh"
   },
   "devDependencies": {
     "@chainlink/contracts": "^0.4.1",
+    "@chainlink/token": "^1.1.0",
     "@nomiclabs/hardhat-ethers": "^2.0.6",
     "@nomiclabs/hardhat-etherscan": "^3.0.4",
     "@nomiclabs/hardhat-waffle": "^2.0.3",
-    "@openzeppelin/contracts": "^4.6.0",
+    "@openzeppelin/contracts": "npm:@openzeppelin/contracts@3.4.1",
```

Comment:
```suggestion
```
I can't find anywhere you use that version of OZ contracts in /batch-nft-reveal. I removed that package locally and everything was working fine.